frontend/src/lib/utils/icp-transactions.utils.ts

```
@@ -270,3 +270,14 @@ export const mapIcpTransactionToUi = ({
     });
   }
 };
+
+export const getTransactionSymbol = (type: AccountTransactionType) => {
+  const positiveTypes = [
+    AccountTransactionType.Receive,
+    AccountTransactionType.Mint,
+    AccountTransactionType.RefundSwap,
+  ];
+
+  if (positiveTypes.includes(type)) return "+";
```

This assumes that any new, currently unknown, type will always be "-".
Better throw an error for an unknown type so we are forced to consider its sign rather than potentially defaulting to a wrong value.